### PR TITLE
Image component

### PR DESCRIPTION
**Pull request by KoushikKintali**
image

### CODE REVIEW

**Review by sanathkumar26700 — APPROVED** [review]

File: components/image/image.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                <div class="components">
+                    <h5>Responsive Image</h5>
+                    <div class="preview-block">
+                        <img src="https://picsum.photos/id/1024/1920/1280" alt="image" class="img-responsive">
+                    </div>
+                </div>
+                <div class="code-block">
+                    <iframe
+                        src="https://carbon.now.sh/embed/yg5bJJK3hpZTUtWtnBSZ"
+                        style="width: 100%; height: 95px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (dev-prabhat) [inline components/image/image.html:71]: remove inline style

File: components/image/image.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                <div class="components">
+                    <h5>Round Image</h5>
+                    <div class="preview-block">
+                        <img src="https://picsum.photos/200" alt="image" class="img-responsive img-round">
+                    </div>
+                </div>
+                <div class="code-block">
+                    <iframe
+                        src="https://carbon.now.sh/embed/L5hkcp6aiIrFrKvgd79R"
+                        style="width: 100%; height: 95px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (dev-prabhat) [inline components/image/image.html:87]: same inline style

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
CSS classes are relevant what it meant to be just remove those inline styles

**Review by MonikaJhingan — APPROVED** [review]
You can avoid using inline styling in upcoming projects rest the code looks good to be merged.

**Pull request merged**